launcher

```diff
@@ -224,6 +225,25 @@ restart() {
     start
 }
 
+gc() {
+    ensure_container_running
+    (
+        loginfo "Stop seafile and start garbage collector"
+        docker exec -it seafile /bin/bash -c "\
+            /opt/seafile/seafile-server-latest/seafile.sh stop; \
+            sleep 10; \
+            /opt/seafile/seafile-server-latest/seaf-gc.sh >> /var/log/gc.log; \
+            sleep 3"
+        loginfo "Garbage collector finished! Restart container..."
+        running=$(docker ps | awk '{ print $1, $(NF) }' | grep " seafile$" | awk '{ print $1 }' || true)
+        if [[ $running == "" ]]; then
+            start
+        else
+            restart
```

Comment:
Why do we need to call start/restart here? We can simply start the server processes again by `seafile.sh start` inside the container.